src/utils/eventListener.js

```diff
@@ -0,0 +1,3 @@
+export default function eventListener(selector, cb) {
+  document.querySelector(selector).onclick = () => cb()
```

Comment:
Доп. обертку не обязательно создавать, `querySelector` потенциально может венруть null
```suggestion
  document.querySelector(selector)?.onclick = cb
```
А так же листенеры могут быть не только `onclick`, можно расширить функционал функции, либо обозвать ее, например, `addClickEventListener`
```suggestion
  document.querySelector(selector)?.addEventListener(eventName, cb)
```